R/g.sib.det.R

```diff
@@ -310,8 +313,18 @@ g.sib.det = function(M, IMP, I, twd = c(-12, 12),
             if (newqqq2 > length(anglez)) newqqq2 = length(anglez)
             # only try to extract SPT again if it is possible to extract a window of more than 23 hour
             if (newqqq2 < length(anglez) & (newqqq2 - newqqq1) > (23*(3600/ws3)) ) {
+              # Recalculate nonwear percentage for new window (6pm to 6pm)
+              nonwear_percentage = (length(which(invalid[newqqq1:newqqq2] == 1)) /  (newqqq2 - newqqq1 + 1)) * 100
+              guider_to_use = 1
+              if (params_sleep[["HASPT.algo"]][1] == "NotWorn" &&
+                  nonwear_percentage < 25 &&
+                  length(params_sleep[["HASPT.algo"]]) == 2) {
```

Comment:
This if-statement is the same as the if statement earlier in this function https://github.com/wadpac/GGIR/blob/master/R/g.sib.det.R#L253-L256, so in the future we will have to make sure that they stay the same. 

How about moving them to a local function and calling the function in both places? Another option is to add a comment in both places to remind ourselves that the condition occurs twice in this function and needs to remain the same.